pkgs/tools/X11/warpd/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, fetchFromGitHub, stdenv, xorg }:
+
+stdenv.mkDerivation rec {
+  name = "warpd";
+  version = "unstable-2022-05-01";
+
+  src = fetchFromGitHub {
+    owner = "rvaiya";
+    repo = name;
+    rev = "133b8a2bd99359baa826c424f90afe3d93434527";
+    sha256 = "sha256-7xPlRn+fMJaaAYHrn6JzcEV771I6wgakd9zetf+vic4=";
+  };
+
+  buildInputs = with xorg; [
+    libXi
+    libXinerama
+    libXft
+    libXfixes
+    libXtst
+    libX11
+    libXext
+  ];
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/man/man1}
+    install -m644 warpd.1.gz $out/share/man/man1/
```

Comment:
`installManPage` which is part of `installShellFiles` could help here.

Reply:
OK, add `installManPage`